src/dotnet/Api/Chat/CopiedChat.cs

```diff
@@ -0,0 +1,33 @@
+using ActualLab.Versioning;
+using MemoryPack;
+
+namespace ActualChat.Chat;
+
+[DataContract, MemoryPackable(GenerateType.VersionTolerant)]
+public partial record CopiedChat(
```

Comment:
ChatCopyState?

Reply:
I can rename it. Should I also rename DbCopiedChat. I tried to avoid general used suffixes.

Reply:
Renamed